testing/kfctl/kf_is_ready_test.py

```diff
@@ -29,6 +30,20 @@ def test_kf_is_ready(namespace, use_basic_auth, use_istio):
 
   util.load_kube_config()
 
+  # If a checklist is provided, use that instead.
+  if checklist:
+    with open(checklist) as f:
+      items = yaml.safe_load(f)
+    for item in items:
+      if item["kind"] == "Deployment":
+        util.wait_for_deployment(api_client, item["namespace"], item["name"], 10)
```

Comment:
too bad we can't check status conditions for these

Reply:
Is this more desirable  over checking that replicas == readyReplicas that the code does right now?
If yes, we could improve the util library in kubeflow/testing.

Reply:
i've seen crashloopbackoff in a number of scenarios for the pods

no intent to hold up your PR, opening an issue is perhaps a better idea,
where applications and events could be checked

Feel free to open an issue otherwise I can

thanks yannis